apollo-backend/src/main/java/io/logz/apollo/deployment/DeploymentHandler.java

```diff
@@ -168,6 +176,9 @@ public Deployment addDeployment(int environmentId, int serviceId, int deployable
 
             logger.info("All checks passed. Running deployment");
             deploymentDao.addDeployment(newDeployment);
+
+            apolloNotifications.notify(Deployment.DeploymentStatus.STARTED, newDeployment);
```

Comment:
Should be inside startDeployment method in KubernetesHandler 
after `deployment.setStatus(Deployment.DeploymentStatus.STARTED);` (103)